src/inventory.rs

```diff
@@ -513,6 +506,27 @@ impl NodeVirtualMachine {
         }
         node_vms
     }
+
+    pub fn node_count(&self) -> usize {
+        self.node_listen_addresses.len()
+    }
```

Comment:
Now that we store >1 listen address per node, we cannot get the node_count from the # of listen addresses.

Reply:
Can we add a new filed to each `NodeVirtualMachine` called as `node_count` that would just get the value from `NodeServiceData::number`. I think that would be the correct and solid approach.